lib/streamlit/runtime/caching/cache_utils.py

```diff
@@ -253,6 +255,21 @@ def get_or_create_cached_value():
                 try:
                     cache.write_result(value_key, return_value, messages)
                 except TypeError:
+                    if type_util.is_type(
+                        return_value, "snowflake.snowpark.dataframe.DataFrame"
+                    ):
+
+                        class UnevaluatedDataFrameError(StreamlitAPIException):
+                            def __init__(self):
+                                StreamlitAPIException.__init__(
```

Comment:
I think you should use `super().__init__()` here ([more information](https://stackoverflow.com/a/27134600))

Reply:
Fixed it 👍